src/locales/en/battler-tags.json

```
@@ -74,5 +74,8 @@
   "substituteOnAdd": "{{pokemonNameWithAffix}} put in a substitute!",
   "substituteOnHit": "The substitute took damage for {{pokemonNameWithAffix}}!",
   "substituteOnRemove": "{{pokemonNameWithAffix}}'s substitute faded!",
-  "autotomizeOnAdd": "{{pokemonNameWithAffix}} became nimble!"
+  "tormentOnAdd": "{{pokemonNameWithAffix}} was subjected to torment!",
+  "tauntOnAdd": "{{pokemonNameWithAffix}} fell for the taunt!",
+  "imprisonOnAdd": "{{pokemonNameWithAffix}} sealed the opponents move(s)!",
+  "autotomizeOnAdd": "{{pokemonNameWIthAffix}} became nimble!"
```

```suggestion
  "autotomizeOnAdd": "{{pokemonNameWithAffix}} became nimble!"
```
Might as well fix Temp's mistake while you're here :plains: